Update README.md (#169)
DataDog/serverless-plugin-datadog: README.md

[![License](https://img.shields.io/badge/license-Apache--2.0-blue)](https://github.com/DataDog/serverless-plugin-datadog/blob/master/LICENSE)

Datadog recommends the Serverless Framework Plugin for developers using the Serverless Framework to deploy their serverless applications.
The plugin automatically enables instrumentation for your Python and Node.js applications to collect metrics, traces, and logs by:

- Installing the Datadog Lambda library to your Lambda functions as a Lambda layer.

> - Installing the Datadog Lambda Extension to your Lambda functions as a Lambda layer (`addExtension`) or subscriping the Datadog Forwarder to your Lambda functions' log groups (`forwarderArn`).

Reviewer: ```suggestion
- Installing the Datadog Lambda Extension to your Lambda functions as a Lambda layer (`addExtension`) or subscribing the Datadog Forwarder to your Lambda functions' log groups (`forwarderArn`).
```

- Making the required configuration changes, such as adding environment variables, to your Lambda functions.

## Getting started

To quickly get started, follow the installation instructions for [Python][1] or [Node.js][2], and view your function's enhanced metrics, traces, and logs in Datadog. These instructions will get you a basic working setup.

## Upgrade

> Each version of the plugin is published with a [specific set of versions of the Datadog Lambda layers](/src/layers.json). To pick up new features and bug fixes provided by the latest versions of Datadog Lambda layers, you need to upgrade the serverless framework plugin. Be sure to test the new version before applying on your production applications.

Reviewer: ```suggestion
Each version of the plugin is published with a [specific set of versions of the Datadog Lambda layers](`/src/layers.json`). To pick up new features and bug fixes provided by the latest versions of Datadog Lambda layers, upgrade the serverless framework plugin. Test the new version before applying it on your production applications.
```
